udap_security_test_kit.gemspec

```diff
@@ -9,8 +9,8 @@ Gem::Specification.new do |spec|
   spec.description   = 'UDAP Security IG Test Kit'
   spec.homepage      = 'https://github.com/inferno-framework/udap-security-test-kit'
   spec.license       = 'Apache-2.0'
-  spec.add_runtime_dependency 'inferno_core', '>= 0.4.2'
```

Comment:
Whoops!

Reply:
Wait, why is this changing? I see `spec.add_runtime_dependency` everywhere else.

Reply:
I think this was a Rubocop auto-correction. When I revert it back to the original my editor gives me a blue squiggly line and says:
> Gemspec/AddRuntimeDependency: Use `add_dependency` instead of `add_runtime_dependency`.RuboCop[Gemspec/AddRuntimeDependency](https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecaddruntimedependency)

I saw you approved the PR but from this comment can't tell if you changed your mind, so @Jammjammjamm just confirm if I'm good to go or if you want me to revert to `spec.add_runtime_dependency`!